elastic/kibana
# [ResponseOps] Hide the query delay capability from non-serverless projects for now

Resolves https://github.com/elastic/kibana/issues/169868

## Summary

Hides the query delay modal and sub feature privilege for non-serverless projects until we onboard all rule types


### Checklist



### To verify

- Run a non serverless project and verify that you can't edit the query delay settings in the rules settings modal
- Run a serverless project for search and observability to verify that you can still edit and update the query delay settings
